storagemarket/impl/providerstates/provider_fsm.go

```diff
@@ -24,18 +24,18 @@ var ProviderEvents = fsm.Events{
 			deal.Message = xerrors.Errorf("deal rejected: %w", err).Error()
 			return nil
 		}),
-	fsm.Event(storagemarket.ProviderEventDealAccepted).
-		From(storagemarket.StorageDealValidating).To(storagemarket.StorageDealProposalAccepted),
+	fsm.Event(storagemarket.ProviderEventDataRequested).
```

Comment:
Is ProviderEventDataRequested and ProviderEventWaitingForManualData functionally equivalent at this point? Maybe we should just combine them.

I ask cause I noticed that in the provider_states.ValidateDealProposal, at the end there is an if statement dispatching two different events, but no actual different code run other than a log message. I'd just make these a single case. (waiting for the client to send us data over data transfer is essentially the same as waiting for the client to send us a hard drive)

Reply:
👍